src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    user,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed === true,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user && (
+        <UserInfo key={user.id} user={user} />
+      )}
```

Comment:
why you need key here??
```suggestion
      <h2 className="TodoInfo__title">{title}</h2>
      {user &&  <UserInfo user={user} />}
```

Reply:
I probably didn’t fully understand how and where to use aribut key :((